onelogin/get.go

```diff
@@ -253,6 +304,28 @@ func getDevice(devices []Device) (device *Device, err error) {
 		return
 	}
 
+	if opts.MfaDevice != "" {
+		for _, d := range devices {
+			if d.DeviceType == opts.MfaDevice {
+				device = &d
+				fmt.Printf("MFA device %s found, automatically selecting it.\n", opts.MfaDevice)
+				return
+			}
+		}
+		// If the user requested device is not found, fall through and continue the device selection process.
+		fmt.Printf("MFA device %s not found.\n", opts.MfaDevice)
+	}
+
+	if opts.AutodetectYubiKey {
+		for _, d := range devices {
+			if d.DeviceType == MFADeviceYubicoYubiKey {
```

Comment:
Out of curiosity, if you have more than two yubikeys plugged in, the selection will be random. I mean, why would you. ;-)
But maybe worth noting in the Readme as a limitation.

Reply:
Noted in the readme